docs/learn/learn-parathreads.md

```diff
@@ -95,8 +95,8 @@ There are two sources of compensation for collators:
 
 Collators may be paid in local parathread currency. However, the Relay Chain transacts with the
 {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} native currency only. Collators must
-then submit block candidates with an associated bid in {{ polkadot: DOT :polkadot }}
-{{ kusama: KSM :kusama }}.
+then submit block candidates with an associated bid in {{ polkadot: DOT. :polkadot }}
+{{ kusama: KSM. :kusama }}
```

Comment:
This seems counter intuitive to what I just said above, but here the 2 conditionals are one newlines.  This would cause there to be a space before the period in Polkadot only.  Placing the period inside for both or removing the space between `}} {{` would also fix the formatting issue.